site/dat/docs/JMeter.md

```diff
@@ -18,13 +18,17 @@ modules:
     - jmeter-ftp
     - jpgc-casutg
 ```
-`plugins` option lets you describe list of JMeter plugins you want to use. If `plugins` option isn't found only 
+`force-ctg` allows you to switch off the usage of ConcurrentThreadGroup for jmx script modifications purpose. This group
+provide `steps` execution parameter but requires `Custom Thread Groups` plugin (installed by default)
+
+With `version` parameter you can ask for specific tool version. Default value is `auto`, it means taurus will chose
+appropriate the JMeter version by itself.
```

Comment:
`by itself` is not enough explanation, tell that version will be taken from JMX file header.